### ISSUE TITLE
Change native ad text color when app theme changes (dark/light theme)

### ISSUE THREAD

**Issue by naamapps** [googleads/googleads-mobile-flutter#551]
## Use case

When the theme of the app changes, the text colors of the native ad stays the same because the colors are defined in the xml/xib file of the native ad.
We need a way to change the colors based on the theme of the app, and not let it be hardcoded like it is now.
For example, when the theme is light - the ad shows as expected, but when changing to dark theme - the ad doesn't change the text color to white, so the ad text is invisible.

## Proposal
I'm no expert in native code (kotlin or swift) so I don't know what the implementation could be.
As I said, We need a way to change the native ad colors based on the theme of the app.

**Comment by maheshmnj** [comment]
Hi @naamapps, Thanks for filing the issue. That seems like a fair proposal to be able to configure ads theme, Labeling this issue as a valid feature request.

cc:@jjliu15

**Comment by 24Lathiya** [comment]
set `customOptions` in Native ads.
`customOptions: {
    ‘darkMode’: isDarkMode,
  }`

for android set below code.

`val darkMode = customOptions?.get("darkMode") as Boolean
            headlineView.setTextColor(Color.parseColor( if(darkMode) "#FFFFFF" else "#000000"))`

My solution is not working on the change theme. it is working based on your app theme. I have put it here because if any has an idea regradings to call `customOptions` on change theme mode.

**Comment by MominRaza** [comment]
Any update on this @maheshmnj 
You found any solution @24Lathiya

**Comment by stjfk** [comment]
> set `customOptions` in Native ads. `customOptions: { ‘darkMode’: isDarkMode, }`
> 
> for android set below code.
> 
> `val darkMode = customOptions?.get("darkMode") as Boolean headlineView.setTextColor(Color.parseColor( if(darkMode) "#FFFFFF" else "#000000"))`
> 
> `customOptions` call only once on load `NativeAd` that's why my solution is not working on the change theme. it is working based on your app's launch time theme(Dark/Light). I have put it here because if any has an idea regradings to call `customOptions` every time on change theme mode. If it is possible then the text color will change immediately on changing theme.
> 
> @maheshmnj @naamapps

this code works when i dispose ad and send request again

but I don't know how to use customoptions on swift side

**Comment by huycozy** [comment]
We’re closing this issue due to inactivity. If you’re still impacted, please create a new issue via the [Developer Forum](https://groups.google.com/forum/#!categories/google-admob-ads-sdk).
